src/libsyncengine/syncpal/syncpal.cpp

```diff
@@ -489,7 +489,7 @@ bool SyncPal::wipeVirtualFiles() {
 
 bool SyncPal::wipeOldPlaceholders() {
     LOG_SYNCPAL_INFO(_logger, "Wiping old placeholders files");
-    VirtualFilesCleaner virtualFileCleaner(localPath());
+    VirtualFilesCleaner virtualFileCleaner(localPath(), syncDbId());
```

Comment:
As discussed offline, the goal is to make sure that the created object is full fledged `VirtualFilesCleaner`  so that any caller can safely rely on its data members and methods.
As you rightly pointed out, this is incomplete, since similar changes should apply to `_vfsStatus` and `_vfsClearFileAttributes`  to make sure every member is properly set.

To be fully addressed in the context of https://infomaniak.atlassian.net/browse/KDESKTOP-1238.